Makefile

```diff
@@ -53,22 +53,18 @@ APPLICATION_VERSION_REDHAT = $(shell echo $(APPLICATION_VERSION) | tr "-" "~")
 # Release type
 # ---------------------------------------------------------------------
 
-# Add the current commit to the version if release type is "snapshot"
-RELEASE_TYPE ?= snapshot
 PACKAGE_JSON_VERSION = $(shell jq -r '.version' package.json)
-ifeq ($(RELEASE_TYPE),production)
+
+ifdef PRERELEASE
```

Comment:
I think it'd be better to invert this, so that `prerelease` is the default-mode (just like `snapshot` used to be the default-mode), and you have to do "something special" (i.e. setting a make-variable) in order to get `regular` builds?